### PR TITLE
LPD-43566 migrate some Categories.testcase to integration tests (Vocabulary trim test)

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
LPD-43566 Migrate Vocabulary character limit to integration tests

# What is this trying to solve?

https://liferay.atlassian.net/browse/LPD-43566

We want to test that the user can not try to add a vocabulary with more than the limit of the name field.

# How am I fixing it?

Like on the frontend part, we trim the titles (all the titles) to the maxlength of the column

# How can you verify that it works?

Run the included automated tests.

# Commits

- **LPD-43566 use services instead of Utils**
- **LPD-43566 trim all the titles on the vocabulary to match with the name column**
- **LPD-43566 add test to check the vocabulary trimmed**

### CODE REVIEW

File: modules/apps/asset/asset-categories-test/src/testIntegration/java/com/liferay/asset/categories/service/test/AssetVocabularyServiceTest.java [hunk 1]
```diff
@@ -549,9 +601,25 @@ protected int searchCount() throws Exception {
...
 
+	private void _testAssetCategoryLongTitlesAreTrimmed(
+		AssetVocabulary assetVocabulary, String title) {
+
+		Map<Locale, String> titleMap = assetVocabulary.getTitleMap();
+
+		for (Map.Entry<Locale, String> entry : titleMap.entrySet()) {
+			Assert.assertEquals(title, entry.getValue());
+		}
+	}
```
Comment (adolfopa) [inline modules/apps/asset/asset-categories-test/src/testIntegration/java/com/liferay/asset/categories/service/test/AssetVocabularyServiceTest.java:612]: Can you rename this to `_assertAssetCategory...`? This looks like an assertion method, not a full blown test.

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/asset/asset-categories-test/src/testIntegration/java/com/liferay/asset/categories/service/test/AssetVocabularyServiceTest.java (hunk 1 above)
Comment (AliciaGarciaGarcia) [reply modules/apps/asset/asset-categories-test/src/testIntegration/java/com/liferay/asset/categories/service/test/AssetVocabularyServiceTest.java]: done!

Thanks!

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by adolfopa** [comment]
ci:reevaluate:1377401_19099

**Comment by AliciaGarciaGarcia** [comment]
rebase with master.

**Comment by AliciaGarciaGarcia** [comment]
ci:test:stable

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:stop

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:stable

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:report:88657982

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:reevaluate:1347401_7028

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
rebased with master due merge conflicts

**Comment by AliciaGarciaGarcia** [comment]
ci:reevaluate:1307401_10639

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:forward:force

**Pull request closed without merging**